{"name": "test_mis_login[testid-testpwd123-管理员]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <scripts.tset03_mis_login.TestMisLogin object at 0x000001B74548BBE0>\nusername = 'testid', psw = 'testpwd123', expect = '管理员'\n\n    @pytest.mark.parametrize(\"username,psw, expect\", read_yaml(\"mis_login.yaml\"))\n    def test_mis_login(self, username, psw, expect):\n>       self.mis_login.page_mis_login(username, psw)\n\ntset03_mis_login.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\page\\page_mis_login.py:24: in page_mis_login\n    self.mis_input_username(username)\n..\\page\\page_mis_login.py:8: in mis_input_username\n    self.base_input(page.mis_username, username)\n..\\base\\base.py:22: in base_input\n    el = self.base_find(loc)\n..\\base\\base.py:17: in base_find\n    return WebDriverWait(self.driver, timeout=timeout, poll_frequency= poll_frequency).until(lambda x: x.find_element(*loc))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ff0edf16d15fdf385b344a6ba2e3c16c\")>\nmethod = <function Base.base_find.<locals>.<lambda> at 0x000001B7450F1160>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "ec0ea2bc-88d0-4433-b24c-e30c6a6ef3b8-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'testid'"}, {"name": "psw", "value": "'testpwd123'"}, {"name": "expect", "value": "'管理员'"}], "start": 1616935709733, "stop": 1616935741020, "uuid": "67859718-6324-4144-a323-1f6253bfeff7", "historyId": "9e7b580274db42bc7b22493eb9c880cd", "testCaseId": "53082eed8f5a39d10c76859b2ad45b68", "fullName": "scripts.tset03_mis_login.TestMisLogin#test_mis_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "tset03_mis_login"}, {"name": "subSuite", "value": "TestMisLogin"}, {"name": "host", "value": "fanxx"}, {"name": "thread", "value": "4836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.tset03_mis_login"}]}